src/common/btree.c

```diff
@@ -638,7 +647,12 @@ btr_node_free(struct btr_context *tcx, umem_off_t nd_off)
 static int
 btr_node_tx_add(struct btr_context *tcx, umem_off_t nd_off)
 {
-	return umem_tx_add(btr_umm(tcx), nd_off, btr_node_size(tcx));
+	struct btr_node *nd = btr_off2ptr(tcx, nd_off);
```

Comment:
(style)  'nd' may be misspelled - perhaps 'and'?
(style)  'nd' may be misspelled - perhaps 'and'?